src/com/lftechnology/java/training/krishna/collections/generics/ProductDemo.java

```diff
@@ -0,0 +1,46 @@
+
+package com.lftechnology.java.training.krishna.collections.generics;
+
+/**
+ * Create a generic class Product with a display method and instantiate an object of Product using different data types.
+ * Eg : Product<String,String> , Product<String,Integer>
+ */
+/**
+ * This program is used to instantiate an object of Product using different data
+ * types.
+ *
+ * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+ * @version 1.0
+ */
+public class ProductDemo {
+
+	// constructor
+	private ProductDemo() {
+
+	}
+
+	public static void main(String[] args) {
+
+		Product<String, String> strObject =
+			new Product<String, String>("Brick", "Stone");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Brick" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)